build.sbt

```diff
@@ -124,6 +124,7 @@ val otherDeps = Seq(
   "com.github.pjfanning"            %% "jackson-scala-reflect-extensions" % V.jackson,
   "com.fasterxml.jackson.dataformat" % "jackson-dataformat-csv"           % V.jackson,
   "com.fasterxml.jackson.datatype"   % "jackson-datatype-jsr310"          % V.jackson,
+  "com.fasterxml.jackson.core"       % "jackson-databind"                 % V.jackson,
```

Comment:
This line is unnecessary, as the other jackson libraries already depend on core and bring it in.